pkg/operator/controllers/guardrails/policies/gktemplates-src/library/common.rego

```diff
@@ -139,4 +143,127 @@ privileged_ns = {
   "openshift-sdn",
   "openshift-service-ca",
   "openshift-service-ca-operator"
-}
\ No newline at end of file
+}
+
+critical_crds = {
```

Comment:
I'm a bit hesitate to add long list to common.rego, as it will appear on all templates that use common lib, feel like it is probably a little better to keep it in the template src file until another policy wants to invoke it, wdyt?